plugins/node/opentelemetry-instrumentation-express/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@opentelemetry/instrumentation-express",
   "version": "0.38.0",
-  "description": "OpenTelemetry express automatic instrumentation package.",
+  "description": "OpenTelemetry instrumentation for `express` http framework",
```

```suggestion
  "description": "OpenTelemetry instrumentation for `express` http web application framework",
```
noticed we had web application written in for koa so adding here too! 😄